bit_manipulation/decimal_to_binary.py

```diff
@@ -0,0 +1,43 @@
+def decimal_to_binary(number: int) -> str:
```

Comment:
number need not be `int`, it can be `float` too since you have named your function as `decimal_to_binary`.  
1 and 1.5 are both decimal numbers because 1 can also be written as 1.0 or 0.01 * 10**2. 
if you are confused about decimal numbers check this [Wikipedia link](https://en.wikipedia.org/wiki/Decimal)